src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java

```diff
@@ -94,7 +99,7 @@ public class PrlAppsConstants {
     public static final String FINAL_HINT = "FINAL";
     public static final String DRAFT_HINT = "DRAFT";
     public static final String DOCUMENT_COVER_SHEET_HINT = "DOC_COVER_SHEET";
-    public static final String DOCUMENT_C7_BLANK_HINT = "DOCUMENT_C7_BLANK";
+    public static final String DOCUMENT_C7_BLANK_HINT = "DOCUMENT_C7_DRAFT";
```

Comment:
change the constant name - blank -> draft

Reply:
resolved